src/server/android/backendAdb.ts

```diff
@@ -111,7 +111,7 @@ class BufferedSocketWrapper extends EventEmitter implements SocketBackend {
     this._socket.on('data', data => {
       debug('pw:adb:data')(data.toString());
       if (this._isSocket) {
-        this.emit('data', data);
+        this.emit('data', data.toString('base64'));
```

Comment:
This doesn't sound right, my socket is a buffer socket.

Reply:
good catch, fixed.